### DISCUSSION TITLE
Some strange logic errors related to the netlist generated by OpenFPGA

### DISCUSSION THREAD

**Discussion by Chris202305** [lnis-uofu/OpenFPGA#1354] [Q&A]
@tangxifan  First of all, I would like to express my sincere thanks for your previous answer, which effectively resolved my doubts. I have a new question now. I have written a simple circuit, and the simulation results are correct. This circuit includes some basic logic operations, primarily adding three numbers after left-shifting them by three bits using multiplication. Theoretically, the lower three bits of the result should be 0. However, after processing it through OpenFPGA and simulating the generated netlist, the simulation shows that these three bits are all 1. What could be causing this issue?
<img width="354" alt="1694675991239" src="https://github.com/lnis-uofu/OpenFPGA/assets/142561908/067ef13f-d320-431a-a130-003a046ad805">
<img width="732" alt="1694676401912" src="https://github.com/lnis-uofu/OpenFPGA/assets/142561908/bfd715f8-29a7-4f83-b43b-97d224c9f895">
<img width="789" alt="1694676615084" src="https://github.com/lnis-uofu/OpenFPGA/assets/142561908/d8a3557b-8e78-41d9-8632-cb7c415b7faa">
<img width="720" alt="1694677289642" src="https://github.com/lnis-uofu/OpenFPGA/assets/142561908/e1252067-a7b1-4b60-ad81-9ec96e242a99">

**Comment by tangxifan** [comment]
@Chris202305 Can you try to add the ``--constant_net_method route`` when calling vpr?

https://github.com/lnis-uofu/OpenFPGA/blob/d6457489cffbdc5814d76bd8a4e4161ae859eb0f/openfpga_flow/openfpga_shell_scripts/fix_heterogeneous_device_example_script.openfpga#L3

**Reply by Chris202305** [reply]
"Your suggestion is very effective. I have an additional question. I'm trying to find the detailed information about the delays of some components in the 'report_timing.setup.rpt' file provided by OpenFPGA. Where can I find this section of the file?
